README.md

```diff
@@ -178,6 +178,15 @@ To delete an importer or exporter, select the delete (x) icon.
 ### Downloading an export
 Once your the exporter has run, a download icon will apear on the exporters menu page.
 
+## Testing
+Bulkrax comes with an extensive test suite; you can run the test suite by:
+
+- Cloning this repository
+- Changing into that directory (e.g. `cd /path/to/bulkrax`)
+- Then running the `rake` command
+
+The first time running the tests you may need to run `bin/rails db:migrate RAILS_ENV=test` to ensure that test database is up to date.
```

Comment:
info about the test suite is in the [contributing doc](https://github.com/samvera-labs/bulkrax/blob/main/CONTRIBUTING.md#running-the-spec-suite). I like having mention of testing in the readme, but what do you think about pointing users there instead so that the information is in one place? less chance for divergence and keeps it DRY.